lib/internal/tty.js

```diff
@@ -53,6 +53,7 @@ const TERM_ENVS = {
   'konsole': COLORS_16,
   'kterm': COLORS_16,
   'mlterm': COLORS_16,
+  'mosh': COLORS_256,
```

Comment:
It seems like mosh has true color support?

Reply:
Hi @BridgeAR Mosh has xterm-256 support. It doesn't have truecolor support. I took a look in mosh repository and found that it has xterm-256 support and not truecolor. So shouldn't it be COLORS_256? Or what is the value for xterm-256?

Reply:
mosh gained truecolor support a few years ago.

Reply:
Hi @devsnek yes I read that it is now supporting truecolor. So is my commit worth merging to the master branch? Or should it be COLORS_16?

Reply:
you should change it to `COLORS_16m`

Reply:
I have corrected the mistake